Core/Src/sensors/APPS.c

```diff
@@ -48,8 +47,8 @@ void APPS_read(void* argument) {
         bse         = ADC_DMA_buffer[BRAKE_E];
 
         // valores de referencia e parametros para o calculo da porcentagem
-        const static apps_ref apps1_ref = {APPS1_REF, APPS1_FIX_MUL, APPS1_FIX_ADD};
-        const static apps_ref apps2_ref = {APPS2_REF, APPS2_FIX_MUL, APPS2_FIX_ADD};
+        const static apps_ref apps1_ref = {APPS1_DEADZONE, APPS1_ADJUST};
```

Comment:
Fazer com a sugestão em cima, mas colocar qual a variavel pra melhorar legibilidade
```suggestion
        const static apps_ref apps1_ref = {.deadzone_limits = APPS1_DEADZONE, .adjust_parameters = APPS1_ADJUST};
```